### ISSUE TITLE
Creating issues to update alt texts on Join Us page

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#3858]
### Dependency
- [x] https://github.com/hackforla/website/issues/3852
- [x] https://github.com/hackforla/website/issues/3856
- [x] https://github.com/hackforla/website/issues/3851
- [x] https://github.com/hackforla/website/issues/3853
- [x] https://github.com/hackforla/website/issues/3854
- [x] https://github.com/hackforla/website/issues/3855
- [x] https://github.com/hackforla/website/issues/3857

### Overview
We need to ensure that alt text for images adhere to Web Content Accessibility Guidelines (WCAG). For this epic, we will write issues to change the alt text for the images in the [SiteMap page](https://www.hackforla.org/join) which will then be added to dependency items above.

### Details
Designers and PMs identified that some alt texts in the [Join Us page](https://www.hackforla.org/sitemap/) should be changed because they are either inaccurate or too long. Ideally, alt text should be descriptive but short and succinct. According to the WCAG page, an accompanying alt text should be short. Additionally, the Functional Accessibility Evaluator (FAE) recommends for alt text to be no more than 100 characters.

### Action Items
- [x] This issue should only be assigned to a lead.
- [x] Lead would guide other developers on creating issues using the following steps (These steps would be recurring for each issue that would be created and this action item should stay unchecked until all the issues have been created):
    - Check the following [Google spreadsheet tab](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=390597843) to track the issue creation progress (there are 7 issues in total that need to be made)
    - Choose a row for which issue needs to be created and add your name in the Issue writer column. (If multiple team members creating issues simultaneously, let other team members know which row you would be working on, so there are no duplicate issues).  See column H to know if an issue has been made already
    - Create a new issue with steps to change the alt text for an image following the sample issue below as well as using [How to create issues](https://github.com/hackforla/website/wiki/How-to-create-issues) wiki.
    - Update the issue link in the [Google spreadsheet tab](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=390597843)
- [x] Add the issue numbers of all the issues that have been created to the dependency section above.
- [x] If all the issues for the spreadsheet tab have been created, add a dependency label and move this issue to icebox column.
- [x] If all the dependency issues are closed, remove the dependency label, move this issue from ice box to In progress column and close this issue.

### Resources/Instructions
- [Join Us page](https://www.hackforla.org/join)
- [Google spreadsheet tab](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=390597843) to track issue creation
- [How to create issues](https://github.com/hackforla/website/wiki/How-to-create-issues) wiki

**Comment by ExperimentsInHonesty** [comment]
thanks @d-perez-8 for making the issues

**Comment by MattPereira** [comment]
Thanks @ExperimentsInHonesty for the bulk issue creation tutorial

**Comment by ExperimentsInHonesty** [comment]
I moved all the issues into the prioritized backlog
